CONTRIBUTING.md

```diff
@@ -21,34 +21,37 @@ Please keep in mind that CodeceptJS have **unified API** for WebDriverIO, Appium
 
 *Whenever a new method or new behavior is added it should be documented in a docblock. Valid JS-example is required! Do **not edit** `docs/helpers/`, those files are generated from docblocks in corresponding helpers! *
 
-Working test is highly appreciated. To run the test suite you need: 
+Working test is highly appreciated. To run the test suite you need:
 
-* selenium server + chromedriver
-* PHP installed
+*   selenium server + chromedriver
+*   PHP installed
```

Comment:
could you revert formatting rule to one space after `*`? I prefer to write it this way